metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/BaseJcrProvider.java

```diff
@@ -437,11 +489,11 @@ protected String sanitizeSystemName(String systemName) {
     protected String sanitizeTitle(String title) {
         return StringEscapeUtils.escapeJava(title);
     }
-    
+
     protected void appendJoins(StringBuilder bldr, String filter) {
         // No joins by default.  Subclasses should override to add any joins needed for sorting and/or filtering.
     }
-    
+
     protected void appendJoins(StringBuilder bldr, Pageable pageable, String filter) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "pageable". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)